tests/test_hoomd.py

```diff
@@ -109,6 +110,10 @@ def test_fallback(tmp_path):
     snap0.configuration.box = [4,5,6,1.0,0.5,0.25];
     snap0.particles.N = 2;
     snap0.particles.types = ['A', 'B', 'C']
+    snap0.particles.type_shapes = [
```

Comment:
Not that it matters too much, but there seems to be two particles and three types.

Reply:
Got it. Thanks for the clarification.